### PR TITLE
disable spec because of selenium/FF35 incompatibility

### PR DESCRIPTION
**Pull request by ulferts**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://fix-temp-disable-query-menu-item-spec.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://fix-temp-disable-query-menu-item-spec.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: spec/features/menu_items/query_menu_item_spec.rb [hunk 1]
```diff
@@ -77,7 +77,10 @@ def check(input_name)
     let!(:menu_item_a) { FactoryGirl.create :query_menu_item, query: query_a }
     let!(:menu_item_b) { FactoryGirl.create :query_menu_item, query: query_b }
 
-    it 'can be renamed and the menu item is sorted', js: true do
+    # disabling until selenium problems with fill_in
+    # on FF35 are resolved.
+    # https://code.google.com/p/selenium/issues/detail?id=8387
+    xit 'can be renamed and the menu item is sorted', js: true do
       new_name = 'aaaaa'
```
Comment (myabc) [inline spec/features/menu_items/query_menu_item_spec.rb:84]: @ulferts I came up with this alternative. The advantage here is that this is (somewhat) self-documenting.

``` diff
diff --git a/spec/features/menu_items/query_menu_item_spec.rb b/spec/features/menu_items/query_menu_item_spec.rb
index f633bac..09deca0 100644
--- a/spec/features/menu_items/query_menu_item_spec.rb
+++ b/spec/features/menu_items/query_menu_item_spec.rb
@@ -77,7 +77,14 @@ feature 'Query menu items' do
     let!(:menu_item_a) { FactoryGirl.create :query_menu_item, query: query_a }
     let!(:menu_item_b) { FactoryGirl.create :query_menu_item, query: query_b }

+    def browser
+      page.driver.browser.capabilities
+    end
+
     it 'can be renamed and the menu item is sorted', js: true do
+
+      skip('cannot be run', if: browser[:browser_name] == 'firefox' && browser[:version].to_i == 35)
+
       new_name = 'aaaaa'

       visit project_work_packages_path(project, query_id: query_b.id)
```

Comment (ulferts) [inline spec/features/menu_items/query_menu_item_spec.rb:84]: @myabc I like it :+1:

**Comment by ulferts** [comment]
@myabc I adapted it according to your idea

**Comment by myabc** [comment]
cool! :+1:

**Pull request merged**